packages/identity-service/src/routes/authentication.js

```diff
@@ -62,7 +62,7 @@ module.exports = function (app) {
               })
             } catch (err) {
               // keep address as null for future user recovery
-              req.logger.error('Error recovering users signed address', err)
+              req.logger.error(err, 'Error recovering users signed address')
```

Comment:
why swap these args?

Reply:
It's a pino thing:
https://github.com/pinojs/pino/blob/main/docs/api.md#mergingobject
> If the object is of type Error, it is wrapped in an object containing a property err ({ err: mergingObject }). This allows for a unified error handling flow.

vs the latter = interpolation values